packages/grafana-data/src/types/dataLink.ts

```diff
@@ -18,6 +39,10 @@ export interface LinkModel<T> {
   title: string;
   target: LinkTarget;
   origin: T;
+
+  // If exists, call this when clicked
+  // with context it is: React.MouseEvent<HTMLAnchorElement>
+  onClick?: (e: any) => void;
```

Comment:
reason for the any type?

Reply:
Unknown what the event source is - angular it is the raw mouse event.  React it is synthetic event (and no react in grafana/data yet)